File: app/src/main/java/com/nickrankin/traktapp/api/services/trakt/model/Person.java
package com.nickrankin.traktapp.api.services.trakt.model;

import com.uwetrottmann.trakt5.entities.PersonIds;

import org.threeten.bp.LocalDate;

public class Person {
    public String name;
    public PersonIds ids;

    // extended info
    public String biography;
    public LocalDate birthday;
    public LocalDate death;
    public String birthplace;
    public String homepage;
    public Boolean series_regular;

}

File: app/src/main/java/com/nickrankin/traktapp/dao/credits/model/Person.kt
package com.nickrankin.traktapp.dao.credits.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import org.threeten.bp.LocalDate
import org.threeten.bp.OffsetDateTime

@Entity(tableName = "people")
data class Person(
    @PrimaryKey val trakt_id: Int,
    val tmdb_id: Int?,
    val imdb_id: String?,
    val name: String,
    val biography: String?,
    val birthplace: String?,
    val birthday: LocalDate?,
    val death: LocalDate?,
    val homepage: String?,
    val picture_path: String?
)